exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -287,9 +288,10 @@ func (p *Poloniex) UpdateOrderbook(currencyPair currency.Pair, assetType asset.I
 		return orderBook, err
 	}
 
-	for _, x := range p.GetEnabledPairs(assetType) {
-		currency := p.FormatExchangeCurrency(x, assetType).String()
-		data, ok := orderbookNew.Data[currency]
+	enabledPairs := p.GetEnabledPairs(assetType)
+	for i := range enabledPairs {
+		curr := p.FormatExchangeCurrency(enabledPairs[i], assetType).String()
```

Comment:
Since we're not using curr again after the map lookup, can just call p.FormatExchangeCurrency directly